package exececao;

public class CheckNotCheck {

	public static void main(String[] args) {
		
		geraErro();
		geraErro2();
		System.out.println("FIM");
	}
	
	static void geraErro() {
		throw new RuntimeException ("Ocorreu um erro");
	}
	
	static void geraErro2() throws Exception{
	   throw new Exception("Erro de excecao");
	}
}
